.eslintrc.json

```diff
@@ -0,0 +1,145 @@
+{
+  "root": true,
+  "plugins": [
+    "@typescript-eslint",
+    "import"
+  ],
+  "parser": "@typescript-eslint/parser",
+  // "parser": "vue-eslint-parser",
```

Comment:
Please remove all comments for .eslintrc file.
I think they cant help with further development.